mslib/msui/updater.py

```diff
@@ -40,19 +42,26 @@ def __init__(self, parent=None):
         self.setupUi(self)
         self.hide()
         self.labelVersion.setText(f"Newest Version: {__version__}")
-        self.updater = Updater()
-        monospace = QtGui.QFont("non-existent")
-        monospace.setStyleHint(QtGui.QFont.Monospace)
-        self.output.setFont(monospace)
-        self.updater.on_log_update.connect(lambda s: (self.output.insertPlainText(s),
-                                                      self.output.verticalScrollBar().setSliderPosition(
-                                                          self.output.verticalScrollBar().maximum())))
-        self.updater.on_status_update.connect(self.statusLabel.setText)
-        self.updater.on_update_available.connect(self.notify_on_update)
-        self.updater.on_update_finished.connect(lambda: self.btRestart.setEnabled(True))
-        self.btUpdate.clicked.connect(lambda: (self.updater.update_mss(), self.btUpdate.setEnabled(False)))
-        self.btRestart.clicked.connect(self.updater._restart_msui)
-        self.updater.run()
+        self.updater = None
+        try:
+            self.updater = Updater()
+        except ModuleNotFoundError as ex:
+            # https://github.com/Open-MSS/MSS/issues/1409#issuecomment-1200946622
+            logging.error("unexpected error in updater: %s %s in version: %s", type(ex), ex, __version__)
```

Comment:
The updater is already on startup called. 
As soon as we know other problems, we can issue a tutorial here. The problem that led to these lines was already very complex.